.github/workflows/pr-build-live-branch.yml

```
@@ -0,0 +1,123 @@
+name: Build live branch for PR
+
+on:
+  pull_request:
```

May change this (equal to manual triggers) and we will slowly roll out this workflow to all PRs later? 

```suggestion
  workflow_dispatch:
```

---

I agree, manual should be fine to begin with and later we can discuss about creating it for every PR

---

In case we want to switch to a manual trigger, I guess the code will need to be adjusted, as it tries to grab values only available in a `pull_request` context, and also it tries to add a PR comment.

---

Yeah, it turns out that I still need the PR number to inform Jetpack Beta Builder so I think it would be better to use `pull_request` context. 

But more important as discussed in https://github.com/Automattic/woocommerce-payments/pull/5768#discussion_r1138451184, we can bring the benefit right away with the new comment including the link to spin up a new JN site right away.